game.html

```diff
@@ -3,21 +3,37 @@
 (function()
 {
```

Comment:
The functionality looks great, but for readablilities sake, you should follow some standards for indenting and bracket placement. For instance the curly brace above should be moved up like this: function() {